spec/helpers/application_helper_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe ApplicationHelper, type: :helper do
+  context ':format_full_date' do
+    it 'formats a date' do
+      allow_any_instance_of(Date).to receive(:strftime).and_return '45 Septemuary 1978'
+      expect(format_full_date(Date.current)).to eq '45 Septemuary 1978'
```

Comment:
Still unsure about these tests. They only confirm that the methods call `strftime` but not that they actually work as expected. The fact that they pass the right arguments for correct formatting seems at least as important.